src/main/java/com/google/alpollo/model/SongSentiment.java

```diff
@@ -0,0 +1,18 @@
+package com.google.alpollo.model;
+
+/**
+ * Class created to store only sentiment score and magnitude, leaving the other useless properties
+ * of the Sentiment object.
+ * Class is mainly used for converting its instances to JSON strings.
+ */
+public class SongSentiment {
+  // Each sentiment has a score, showing the overall positivity of the text, ranging from -1.0 to 1.0.
```

Comment:
Use javadoc notation (i.e. `/**`) instead of inline comments (i.e. `//`) to document fields as well